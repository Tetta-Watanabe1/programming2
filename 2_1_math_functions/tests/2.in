2.71828 10
