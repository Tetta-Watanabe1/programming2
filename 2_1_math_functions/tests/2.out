1.64872 2.20263e+04 0.41078 0.43429 1.00000 
